symplyphysics/laws/optics/abbe_invariant.py

```diff
@@ -0,0 +1,41 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression,
+    validate_input, validate_output, dimensionless)
+
+# Description
+## The Abbe invariant is an optical characteristic of the refractive curve of the surface
+
+# Law: Q = n * ((1 / d) - (1 / R))
+# Where:
+## n - surface refraction index
```

Comment:
It's not surface refraction index. It's medium refraction index before or after the surface.

Reply:
You're right. I wanted to say: "the refractive index of the optical environment"